### ISSUE TITLE
Symfony serialized process on symfony 2.7.5

### ISSUE THREAD

**Issue by carmelomtb** [symfony/symfony#16110]
I put this url in my browser and press enter
http://localhost:8000/late
than i put this url in my browser and press enter
http://localhost:8000/fabien

I can see "Hello Fabien" only after the "Late" page is rendered

This is my code:

	/**
	 * @Route("/late", name="_late" )
    */
    public function lateAction()
    {				
	sleep(10);
       return $this->render('AcmeDemoBundle:Default:index.html.twig', array('name' => 'late')); 
    }
	
	/**
	 * @Route("/fabien", name="_fabien" )
    */
    public function fabienAction()
    {				
		return $this->render('AcmeDemoBundle:Default:index.html.twig', array('name' => 'fabien'));
    }

**Comment by Tobion** [comment]
Maybe you are using sessions, which serialize access to avoid race conditions.

**Comment by Tobion** [comment]
I just tried to reproduce the problem. And indeed even when disabling sessions, requests are blocking. But this is only a limitation of the PHP built-in server

> The web server runs a only one single-threaded process, so PHP applications will stall if a request is blocked.

http://php.net/manual/en/features.commandline.webserver.php

**Comment by carmelomtb** [comment]
My system: Mac OS X 10.10, MAMP 3, php 5.5.14, symfony 2.7.5 clear installation (I created a brand new bundle whit the above code in the controller)
I did other 4 experiments:
1) I used the above code in symfony 2.5.9 whit no problem. I saw immediately "Hello Fabien" 
--------------
2) I used the above code in symfony 2.6.11 whit the same problem like symfony 2.7.5. I saw "Hello Fabien" only after the "Late" page was rendered.
3)
File fabien.php:
<?php
    echo 'Hello Fabien';
?>
File late.php:
<?php
    sleep(10);
    echo 'Late';
?>
I put in my browser the url http://localhost/late.php, than i put in a new tab http://localhost/fabien.php. I saw immediately "Hello Fabien"
-------------------------------
4) I use my windows pc to access my mac through the network:
Symfony 2.7.5
First url: http://192.168.1.70/test/web/late
Second url:  http://192.168.1.70/test/web/fabien
I saw immediately "Hello Fabien"
